locales/fr/_excludes.json

```diff
@@ -33,6 +33,7 @@
     "Honduras",
     "Hong Kong",
     "ID",
+    "Iran",
```

Comment:
Il n'y a pas de tag "Iran"

Reply:
Si je me fis au code de contribution du repo, comme la traduction "Iran, Islamic Republic of" est une sous-chaîne de la chaîne originale, elle est considérée non traduite, d'où l'exclusion.

Reply:
cela est plus dû à un bug dans le script des tags non traduits, donc ok j'accepte les deux pays